modules/user/src/main/User.scala

```diff
@@ -112,7 +112,11 @@ object User {
 
   type CredentialCheck = String => Boolean
   case class LoginCandidate(user: User, check: CredentialCheck) {
-    def apply(password: String): Option[User] = check(password) option user
+    def apply(password: String): Option[User] = {
+      val res = check(password)
+      lila.mon.user.auth.result(res)
```

Comment:
pretty sure you need to append '()' to actually increment the monitor

Reply:
nice catch